sentences/gl/_common.yaml

```diff
@@ -47,7 +47,7 @@ lists:
       - in: "f"
         out: "fahrenheit"
 expansion_rules:
-  name: "[o | a] {name}"
+  nome: "[o | a | os | as] {name}"
```

Comment:
My understanding is that keys must remain in english, only values are internationalized.

Reply:
on a second thought, that's not necessary for expansion rules, only for other keys. I'll approve it.

Reply:
Yep I checked other languages and that's indeed the case.